deployment/terraform/create.go

```diff
@@ -408,6 +408,10 @@ func (t *Terraform) displayInfo(output *terraformOutput) {
 	for _, agent := range output.Agents.Value {
 		mlog.Info(agent.Tags.Name + ": " + agent.PublicIP)
 	}
+	if len(output.Agents.Value) > 0 {
+		mlog.Info("Coordinator:" + output.Agents.Value[0].PublicIP)
+		mlog.Info(fmt.Sprintf("To start coordinator, you can use %q command.", "ltctl loadtest start"))
```

Comment:
Maybe I'd change the command to be `go run ./cmd/ltctl loadtest start` since the deployment is currently possible only from the repository and the guide too does suggest that. It's a minor detail but I want to avoid any possible point of ambiguity for the less experienced user.

Reply:
Hmm, you are right. I was thinking for the release. But I think we can do both 😇